### PR TITLE
Display List of Items on the Home page

### PR DESCRIPTION
**Pull request by souad988**
Create Request class to handle the connection with the API.
Create the list of items from data retrieved from the API.
Add styling to the list.
Add pagination to display items 10 by 10.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,26 @@
 import './style.scss';
+import Request from "./modules/request";
+
+const coinsUrl='https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1&sparkline=false';
+const url='https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/QzuepOdb4S7f14QxmbA0/scores';
+const request = new Request(coinsUrl);
+const req= new Request(url);
+const coins=document.querySelector('.coins');
```
Comment (damdafayton) [inline src/index.js]: If this query was in the `addToDom`() , function would have less arguments passed into and code would be easier to read.

File: src/index.html [hunk 2]
```diff
@@ -30,6 +30,21 @@ <h1>Cryptocurrency Prices</h1>
                 <li>circulating_supply</li>
```
Comment (damdafayton) [inline src/index.html:30]: Need to remove `_`

**Review by damdafayton — COMMENTED** [review]
✔️ Nice design

File: src/index.js (hunk 1 above)
Comment (damdafayton) [inline src/index.js]: If this query was in the `addToDom`() , function would have less arguments passed into and code would be easier to read.

Comment (damdafayton) [reply src/index.js]: I mean if this line 
`const coins=document.querySelector('.coins');`
was in below function
there wouldnt be need to pass `ul` as function argument
```
const addToDom=async(request,ul)=>{
    var result=await getData(request);
    ul.innerHtml = '';
    console.log(result);
     result.forEach(element => {
             ul.innerHtml +=`<li>${element.name}</li>`;
             console.log(element.name)
          }); 
}
```

**Review by damdafayton — APPROVED** [review]
Good job.

File: src/index.js (hunk 1 above)
Comment (souad988) [reply src/index.js]: ok but i think i is better to pass it as argument for code reusability if i want to use it with a different ul or element i wont have to create an other function.

**Pull request merged**